clients/big_query/lib/google_api/big_query/v2/model/table_cell.ex

```diff
@@ -20,11 +20,14 @@
 defmodule GoogleApi.BigQuery.V2.Model.TableCell do
   @moduledoc """
   
+
+  ## Attributes
+
+  - v (String.t):  Defaults to: `null`.
   """
 
-  @derive [Poison.Encoder]
   defstruct [
-    
+    :"v"
```

Comment:
This was `type: 'any'` in the google discovery doc. The converter has been updated to handle these.

Reply:
This is the value inside the cell then? And we think handling this as a string will work? I'm not sure what the implications are here, say if it's a nested object and these values are booleans.

Reply:
Basically, we do nothing with the json parsed value. So if there is somehow a nested value, it will be a raw map, list or literal string.